File: beginners/easy/010.read-a-positive-integer-and-calculate-the-sum/206.saadman-galib.readAPositiveIntegerAndCalculateTheSum.cpp
#include<iostream>
using namespace std;

int main(){
    int num, sum = 0;
    while (true){
        cout << "Enter a positive integer: ";
        cin >> num;
        if (num >= 0)
        {
            for (int i = 0; i <= num; i++)
            {
                sum += i;
            }

            cout << "Sum = " << sum << endl;
            break;
        }
    else
        cout << "Sorry to say but the number you have entered is a negative number. Pls enter again." << endl;
        }
    return 0;
}

File: beginners/easy/010.read-a-positive-integer-and-calculate-the-sum/206.saadman-galib.readAPositiveIntegerAndCalculateTheSum.py
i = 0
sum = 0
while True:
    n = int(input("Enter a positive integer: "))
    if n >= 0:
        while i <= n:
            sum += i
            i += 1
        print("Sum =", sum)
        break
    else:
        print("Sorry to say but the number you have entered is a negative number. Pls enter again.")